frontend/public/components/utils/resource-log.jsx

```diff
@@ -116,12 +117,14 @@ export class ResourceLog extends SafetyFirst {
 
   // Handler for websocket onmessage event
   _onMessage(msg){
+    const { linesBehind } = this.state;
     const text = Base64.decode(msg);
```

Comment:
So you get `msg` but are pivoting off `text` which is `Base64.decode(msg)`.  At what point will these be out of sync?  IE,  `msg` is `null`, `undefined`, `''`, something `falsy`, but `text` could still be `truthy` & trigger the code in the `if`?

I'm guessing there is some way this is possible, I just don't see it atm.

Reply:
Hmm, yeah, I think you're right. Just tested this and `Base64.decode(null)` and `Base64.decode(undefined)` will return garbage strings, which would be truthy. I'll fix this.